<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="tree_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="tree_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>398</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="tree_8">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>398</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="tree_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>400</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>ff</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>ff2</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>ff3</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="table_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>새 열</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 행</string>
     </property>
    </column>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>640</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>610</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
